docker-compose.yml

```diff
@@ -24,8 +23,6 @@ services:
     ports:
       - 0.0.0.0:1234:1234
     healthcheck:
-      test: ["CMD", "curl", "-f", "http://localhost"]
+      test: ["CMD", "curl", "-f", "http://localhost:1234"]
     depends_on:
       - redis
-    volumes:
```

Comment:
With this removed, how does docker have any access to the code? I'm not a docker expert, so you're going to have to spell it out for me.

Reply:
If this compose is for dev on local, this volumes is necessary for propergating local changes into docker,
but in production case,  codes are added in Dockerfile [ADD step](https://github.com/mobtimeapp/mobtime/pull/266/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557L5), so this volume is unnecessary.

Reply:
Okay, I think we should have a `docker-compose.yml` for base settings, and `docker-compose.development.yml` and `docker-compose.production.yml` to override for their environments. Is that alright? I think then we can use the volume for development and skip it for production.

Reply:
Ok, `docker-compose.development.yml` uses volume, and `docker-compose.production.yml` does not.

What's the differences between `docker-compose.yml` and `docker-compose.development.yml`?

Reply:
thanks desc, I understood!